scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -175,6 +179,7 @@ def net_interfaces_config(args, nicmodel):
             cloud=args.cloud,
             nodecounter=args.nodecounter,
             nicindex=index,
+            net=get_net_for_nic(args, index),
```

Comment:
The invariant here is `index >= 0`, and `arg.ironicnic` can be -1, so if there are multiple nics both will be for the admin network, is this OK?